flink-connectors/flink-connector-gcp-pubsub/pom.xml

```diff
@@ -56,18 +56,22 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>com.google.cloud</groupId>
+			<artifactId>google-cloud-core</artifactId>
+			<!-- Version is pulled from google-cloud-bom (loaded via the libraries-bom) -->
+		</dependency>
+
 		<dependency>
 			<groupId>com.google.cloud</groupId>
 			<artifactId>google-cloud-pubsub</artifactId>
-			<!-- Version is pulled from google-cloud-bom -->
-			<exclusions>
-				<!-- Exclude an old version of guava that is being pulled
-                in by a transitive dependency of google-api-client -->
-				<exclusion>
-					<groupId>com.google.guava</groupId>
-					<artifactId>guava-jdk5</artifactId>
-				</exclusion>
-			</exclusions>
+			<!-- Version is pulled from google-cloud-bom (loaded via the libraries-bom) -->
+		</dependency>
+
+		<dependency>
+			<groupId>com.google.api.grpc</groupId>
+			<artifactId>grpc-google-cloud-pubsub-v1</artifactId>
+			<!-- Version is pulled from google-cloud-bom (loaded via the libraries-bom) -->
```

Comment:
Why is this dependency needed? Isn't `google-cloud-pubsub` enough?

Reply:
No, this one is really needed. I also looked into this while writing it.
Apparently this connector uses in a few places the com.google.pubsub.v1.SubscriberGrpc which is not included in the dependencies otherwise.

Reply:
Mh, ok :(